client/src/components/WaitingRoom/index.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+import { withFirebase } from '../Firebase';
+import { compose } from 'recompose';
+import { withAuthorization } from '../Session';
+
+class WaitingRoomBase extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      players: [],
+      gameId: '',
+      timeLimit: 10,
```

Comment:
Is there a reason for this default time limit?

Reply:
I believe the default value should be `null` because that's what I put down when creating a game in the database.

Reply:
was under the impression that there needed to be a default value for each state parameter, but since that's not the case there's no need for it